entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityOperationHandlerTests.kt

```diff
@@ -45,7 +49,9 @@ class EntityOperationHandlerTests {
         val jsonLdFile = ClassPathResource("/ngsild/hcmr/HCMR_test_file.json")
         val createdEntitiesIds = arrayListOf("urn:ngsi-ld:Sensor:HCMR-AQUABOX1temperature", "urn:ngsi-ld:Sensor:HCMR-AQUABOX1dissolvedOxygen", "urn:ngsi-ld:Device:HCMR-AQUABOX1")
 
-        every { entityService.exists(any()) } returns false
+        every { validationUtils.getNewEntities(any()) } returns listOf()

```

Comment:
you can avoid this boilerplate code by declaring the mockked bean as relaxed (`@MockkBean(relaxed = true)`), useful when you are not interested by a specific result returned by your mockked bean

Reply:
Thanks for the tip. :+1: